torch/_dynamo/variables/builder.py

```diff
@@ -1074,6 +1075,9 @@ def build_key_value(i, k, v):
                 fake_script_obj,
                 source=self.source,
             )
+        elif isinstance(value, MutableMapping):
```

Comment:
Lets make it `issubclass` instead of `isinstance`. I think there is lot of duck-typing going on with `MutableMapping` such that even `ininstance(dict(), MutableMapping` is `True`.